data/bird_move_global.csv

```diff
@@ -0,0 +1,141 @@
+"latitude","week","global_function","n_indiv","obs_prob","global_scaled_function"
```

Comment:
Do we want to show the side constraint issue for the `bird_move` data set? I thought the figure you produced in #49 summed up the issue with identifying the global term nicely and suggested `t2()` as the method best able to reconstruct the global term, but the narrative currently ignores this example.

I'm torn between keeping things simple and trying to justify the new narrative; there is a lot of text without any supporting references. Could we not just include the figure with a sentence or two to explain that the `t2()` seems to work best because of the way the null and range space parts of the penalty are separated, so that even though side constraints are applied they seem to have less of an effect?

Reply:
I'm okay with putting it in; my only reservation is that the paper's already getting quite long. @dill, as the other participant in the side-constraint issue, what do you think? Should we include the figure in #49 for clarity?

Reply:
it would be nice to have the figure. Agree that the paper is a bit of a monster... Is it possible to export some of this to an appendix? I think we had a chat a while ago about appendicizing some portion of the paper, but I guess we didn't chase that up??